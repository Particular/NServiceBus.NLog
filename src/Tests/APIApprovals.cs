using System.Reflection;
using NUnit.Framework;
using Particular.Approvals;
using PublicApiGenerator;

[TestFixture]
public class APIApprovals
{
    [Test]
    public void Approve()
    {
        var publicApi = Assembly.Load("NServiceBus.NLog").GeneratePublicApi(new ApiGeneratorOptions
        {
            ExcludeAttributes = new[] { "System.Runtime.Versioning.TargetFrameworkAttribute", "System.Reflection.AssemblyMetadataAttribute" }
        });
        Approver.Verify(publicApi);
    }
}