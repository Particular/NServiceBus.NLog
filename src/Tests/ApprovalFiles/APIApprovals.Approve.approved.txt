namespace NServiceBus
{
    [System.Obsolete(@"NServiceBus is now providing support for logging libraries through the Microsoft.Extensions.Logging abstraction. Remove the NServiceBus.NLog package. Install the NServiceBus.Extensions.Logging and NLog.Extensions.Logging packages instead. Will be removed in version 5.0.0.", true)]
    public class NLogFactory : NServiceBus.Logging.LoggingFactoryDefinition
    {
        public NLogFactory() { }
        protected override NServiceBus.Logging.ILoggerFactory GetLoggingFactory() { }
    }
}